shared/utility/support/hostname.cpp

```diff
@@ -0,0 +1,17 @@
+#include "hostname.hpp"
+
+namespace utility::support {
+
+    std::string getHostname() {
+        char* hostname = std::getenv("ROBOT_HOSTNAME");
+
+        if (hostname == nullptr) {
+            char buffer[255];
+            gethostname(buffer, 255);
```

Comment:
Pedanticness I guess
```suggestion
            ::gethostname(buffer, 255);
```